mlir/lib/Dialect/Quant/IR/TypeParser.cpp

```diff
@@ -196,8 +196,9 @@ static ParseResult parseQuantParams(DialectAsmParser &parser, double &scale,
                                     int64_t &zeroPoint) {
   // scale[:zeroPoint]?
   // scale.
-  if (parser.parseFloat(scale))
+  if (parser.parseFloat(scale)) {
```

Comment:
how come you added these braces?

Reply:
this wasn't really necessary, but I compulsively changed it for consistency with our code style https://github.com/intel-innersource/applications.ai.vpu-accelerators.vpux-plugin/blob/develop/src/vpux_compiler/docs/code_style.md#braces as we normally use OTBS in vpux (this is actually not vpux repo but it seems that OTBS is mostly used in the files I've inspected)